release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import '../logic/git.dart';
```

Comment:
this import is out of order.

Add pub.dev/packages/flutter_lints to catch these issues